common.blocks/page/_view/page_view_500.bemtree.js

```diff
@@ -0,0 +1,19 @@
+block('page').mod('view', '500').content()(function() {
```

Comment:
it's just about `page` content, so there's no need to provide doctype, html, head and body tags, just content itself:
```js
block('page').mod('view', '500').content()(function() {
    return [
        {
            tag: 'h1',
            content: 'Внутреняя серверная ошибка'
        },
        {
            tag: 'p',
            content: 'Извините, что-то пошло не так. Сервер не смог обработать ваш запрос.'
        },
        {
            tag: 'p',
            content: 'Пожалуйста, попробуйте еще раз позже.'
        }
    ];
});
```

Also `i18n` should be used.